pygis/docs/e_extraction.md

```diff
@@ -0,0 +1,323 @@
+---
+jupytext:
```

Comment:
use comments from other pull request

Reply:
Done! Changes for e_extraction.md committed/pushed.